subprojects/android-test/test-screenshot/src/main/java/com/avito/android/test/screenshot_test/internal/DeviceDirectoryName.kt

```diff
@@ -0,0 +1,43 @@
+package com.avito.android.test.screenshot_test.internal
+
+import android.content.Context
+import android.util.DisplayMetrics
+import android.view.Display
+import android.view.WindowManager
+import kotlin.math.roundToInt
+
+data class DeviceDirectoryName(
```

Comment:
Please check the classes' visibility and use internal as much as possible.
Any public entity in API makes harder to change a library due to backward compatibility.

Reply:
done